analyses/doublet-detection/exploratory-notebooks/03_compare-benchmark-results.Rmd

```diff
@@ -191,9 +194,10 @@ doublet_df_list <- dataset_names |>
       scuttle::makePerCellDF(sce, use.dimred = "PCA") |>
         tibble::rownames_to_column(var = "barcodes") |>
         dplyr::select(barcodes,
```

Comment:
This is a place where I often find the styler messes up a bit. If any args are on separate lines, everything should be. 
```suggestion
        dplyr::select(
          barcodes,
```